feat: allow `:global` in more places (alternative) (#12560)
sveltejs/svelte: packages/svelte/messages/compile-errors/style.md

> A :global {...} block cannot modify an existing selector

## css_global_block_invalid_modifier_start

> > A :global {...} block at the very beginning cannot be modified by other selectors

Reviewer: We're going to need to rewrite all the ':global {...} block' messages to account for `:global .whatever`

```suggestion
> A `:global` selector can only be modified if it is a descendant of other selectors
```

Author: Either Github played a trick on you or you commented right before I pushed changes in that direction - either way, I adjusted the messages
